web/app/js/components/Debug.jsx

```diff
@@ -0,0 +1,129 @@
+import BaseTable from './BaseTable.jsx';
+import ErrorBanner from './ErrorBanner.jsx';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import React from 'react';
+import Spinner from './util/Spinner.jsx';
+import Typography from '@material-ui/core/Typography';
+import _each from 'lodash/each';
+import _get from 'lodash/get';
+import _reduce from 'lodash/reduce';
+import { apiErrorPropType } from './util/ApiHelpers.jsx';
+import { numericSort } from './util/Utils.js';
+import { withContext } from './util/AppContext.jsx';
+import withREST from './util/withREST.jsx';
+
+const endpointColumns = [
+  {
+    title: "Namespace",
+    dataIndex: "namespace",
+    render: d => <Link to={"/namespaces/" + d.namespace}>{d.namespace}</Link>,
+    sorter: (a, b) => (a.namespace).localeCompare(b.namespace)
+  },
+  {
+    title: "IP",
+    dataIndex: "ip"
+  },
```

Comment:
TIOLI, "IP" is the only column that's not sortable. Would it make sense to add a sorter for it? I think it's also totally fine for it to not be sortable, but in that case I'd recommend removing the sorter from the "Port" column, since it feels pretty similar to this column.